### PR TITLE
fix(DB/Creature): remove mobs from brewfest camp

### PR DESCRIPTION
**Pull request by locus313**
## CHANGES PROPOSED:
-  remove hostile and non-hostile mobs from brewfest camp

## ISSUES ADDRESSED:
- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/3496

## TESTS PERFORMED:
- Applied sql and restart server
- Screenshot of test in https://github.com/azerothcore/azerothcore-wotlk/issues/3496

## HOW TO TEST THE CHANGES:

1. Go to the brewfest camp right outside orgrimmar
2. Check if you seen any hostile or non-hostile mobs within the camp area 

## Target branch(es):
- [x] Master

 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by genion18** [comment]
is this are not better when you makes this creatures in an Event?

Then the event start are the Creature Hidden and then not creature unhidden.

**Comment by Kitzunu** [comment]
No because you assign a creature to an event.
Like

Event X active == Creatue P visible

You cannot do it the other way around

**Comment by genion18** [comment]
ah, okay. is not implemented yet. thanks for this info.

**Comment by Mitradis** [comment]
STOP THIS F/// SHIT PLZ.

INSERT INTO `game_event_creature` (`eventEntry`,`guid`) VALUES
(5,137686),
(-24,21026),
(-24,12379),
(-24,21034),
(-24,7370),
(-24,7369),
(-24,12386),
(-24,12369),
(-24,22473),
(-24,22188),
(-70,21026),
(-70,12379),
(-70,21034),
(-70,7370),
(-70,7369),
(-70,12386),
(-70,12369),
(-70,22473),
(-70,22188);
DELETE FROM `game_event_creature` WHERE `eventEntry` IN (-26,33);

**Comment by Mitradis** [comment]
AND HOW MANY NPCs you deleting like this??? WTF.

**Comment by FrancescoBorzi** [comment]
@Mitradis what? Sorry I don't understand your concern, what minus? Where?

**Comment by locus313** [comment]
@Mitradis Thanks for pointing out about the `game_event_creature` table. But do note that the contributors on this project are all at different skill levels and in many cases are still learning. (that includes me)
@FrancescoBorzi I see in the info here:
https://www.azerothcore.org/wiki/game_event_creature
about he minus option for the `game_event_creature` table.

I ran a new test using the `game_event_creature` table and using the minus does indeed remove the mobs from the brewfest camp area. I will update the pr shortly. Here is a screenshot from the test I just ran:
image

**Comment by FrancescoBorzi** [comment]
Ah ok, now I get it. At first, I didn't see any minus in the PR so I assumed the comment was maybe a mistake.

Thanks @Mitradis for your feedback!

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by genion18** [comment]
And this is what i mean, creature hidden in an event and unhidden where event is end.

**Comment by Si1ker** [comment]
I asked if this could be possible in the past few days @FrancescoBorzi @Kitzunu 
image











On the other hand... @Mitradis  Thanks for the tip and stuff but... (I will say it as you would :) )  why the f*ck you have to come with "STOP THIS F/// SHIT PLZ." and  "Admins seriously, you don't know this?".

Is it impossible for you to say "Hello, I think there is a better way to do this.. etc etc.."

AzerothCore is maintained by people with different skills and knowledge (as it is already said above) with passion for AC, most of the time the 'top devs' are dedicated to private projects where they earn for their work so they avoid to come here and do stuff for free and contribute to this open-source. Every single PR, makes AC evolve, there are bad PRs with good intentions and good PRs that may break something which is fixed then with another PR in the near future.

'The knowledge is welcome, the modals.. simply appreciated.'

**Comment by Kitzunu** [comment]
Actually I did not know of that :)

File: data/sql/updates/pending_db_world/rev_1600739072121853409.sql [hunk 1]
```diff
@@ -0,0 +1,18 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1600739072121853409');
+
+/* brewfest camp hostile mobs */
+DELETE FROM `game_event_creature` WHERE `guid` IN (7370, 22181, 22188, 22473);
+INSERT INTO `game_event_creature` (eventEntry,guid) VALUES
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1600739072121853409.sql:5]:
```suggestion
INSERT INTO `game_event_creature` (`eventEntry`, `guid`) VALUES
```

File: data/sql/updates/pending_db_world/rev_1600739072121853409.sql [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+(-24,7370),
+(-24,22181),
+(-24,22188),
+(-24,22473);
+
+/* brewfest camp non-hostile mobs */
+DELETE FROM `game_event_creature` WHERE `guid` IN (12369, 12372, 21020, 21022, 21026);
+INSERT INTO `game_event_creature` (eventEntry,guid) VALUES
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1600739072121853409.sql:13]:
```suggestion
INSERT INTO `game_event_creature` (`eventEntry`, `guid`) VALUES
```

**Review by Kitzunu — CHANGES_REQUESTED** [review]
Keywords should ALWAYS be within \` \` whatever those quotes are called :P

**Comment by locus313** [comment]
Thanks. I was trying to remember that, but overlooked those ones.

**Pull request merged**